components/server/src/migration/user-to-team-migration-service.ts

```diff
@@ -29,7 +29,7 @@ export class UserToTeamMigrationService {
         if (!this.needsMigration(candidate)) {
             return candidate;
         }
-        return this.redisMutex.using(["migrateUser", candidate.id], 5000, async () => {
+        return this.redisMutex.using(["migrateUser-" + candidate.id], 5000, async () => {
```

Comment:
I wonder why we are appending here, instead of the default case, where we're using the array case? :thinking: 

https://github.com/gitpod-io/gitpod/pull/17736/files#diff-2d1814d4e79d79146e0a3be29f6118d6e93de6bfb60c8f4ce44f93c5a3ccdfbdR79

Reply:
That was wrong usage and is what this PR is all about.